app/src/main/java/org/simple/clinic/bp/history/BloodPressureHistoryScreenInit.kt

```diff
@@ -6,6 +6,6 @@ import org.simple.clinic.mobius.first
 
 class BloodPressureHistoryScreenInit : Init<BloodPressureHistoryScreenModel, BloodPressureHistoryScreenEffect> {
   override fun init(model: BloodPressureHistoryScreenModel): First<BloodPressureHistoryScreenModel, BloodPressureHistoryScreenEffect> {
-    return first(model, LoadBloodPressureHistory(model.patientUuid))
+    return first(model, LoadPatient(model.patientUuid), LoadBloodPressureHistory(model.patientUuid))
```

Comment:
Not necessary to do in this PR, but since the model is being saved and restored, we should also add checks to avoid loading the patient details if the screen is restored and the patient details already exist in the model.

It might make sense to continue loading the BP history even when it is restored because it is more likely to change frequently than the patient details and it is a reactive query. This means we can avoid parcelling them in the model and mark them as `@IgnoredOnParcel`.